app/globals.css

```diff
@@ -13,6 +13,14 @@ h1 {
     @apply font-bold
 }
 
+main {
```

Comment:
Rather than doing styling here, you can do it on the 'main' element itself on the page.tsx file. You wanna keep this global file only for elements that you cannot customise elsewhere.